README.md

```diff
@@ -0,0 +1,219 @@
+# Personal README
+
+__What is this?__
+
+Think of this as documentation for Matthew Root. Its intended purpose is to accelerate your understanding of me as a human and Engineering Manager and to provide insight into what our working relationship will look like. It is __not__ intended as a substitute for getting to know me. Building a relationship, in real life, is one of the most important things we'll do together. This README should be used as a supplement and a guide rather than a replacement for that.
+
+<!-- TOC -->
+
+- [Management Philosophy](#management-philosophy)
+- [Expectations](#expectations)
+    - [What you can expect of me](#what-you-can-expect-of-me)
+    - [What I will expect of you](#what-i-will-expect-of-you)
+    - [Balancing life and work](#balancing-life-and-work)
+- [Communicating With Me](#communicating-with-me)
+    - [Talking](#talking)
+    - [Slack](#slack)
+    - [Email](#email)
+    - [PagerDuty](#pagerduty)
+    - [Mobile phone](#mobile-phone)
+    - [Social media](#social-media)
+- [Bugs & Quirks](#bugs--quirks)
+    - [Direct and intense](#direct-and-intense)
+    - [Fast talker, sporadic rambler](#fast-talker-sporadic-rambler)
+    - [Stubborn](#stubborn)
+    - [Gets bored easily](#gets-bored-easily)
+- [My Story](#my-story)
+    - [Chapter 1: NOVA](#chapter-1-nova)
+    - [Chapter 2: State College](#chapter-2-state-college)
+    - [Chapter 3: Various](#chapter-3-various)
+    - [Chapter 4: San Francisco](#chapter-4-san-francisco)
+- [Personal Interests](#personal-interests)
+- [Want to work together?](#want-to-work-together)
+
+<!-- /TOC -->
+
+## Management Philosophy
+
+The following are the core tenets that provide the foundation for how I prefer to manage. I strive to adhere to these all the time. If you ever find me to be in violation of one of these ideas, please point it out to me. That will help me do better in the future.
+
+- __Measuring stick for success:__ Can someone go out and get a better role after you’ve been their manager? You’ve got to care about the development of people to your core.
+- __Put people and the team first:__ Constantly ask what you can do to be most helpful to individuals or the team? The most helpful thing you can do may be totally different day-to-day, week-to-week, person-to-person, or team-to-team. Default to the assumption that you’re the worst on the team and treat everyone else like they’re more awesome than you. If you’ve hired right, that will be true.
+- __Constant improvement:__ Make this a fundamental value for both individuals and the team itself. Don’t set anything in stone. Don’t assume you’re ever doing something perfect. When you’ve made one improvement, move onto the next thing. Repeat forever.
+- __How to move mountains:__ Give a properly motivated team a high degree of trust and autonomy.
```

Comment:
This is another aspect that is difficult to articulate with more specificity without writing _a lot_. Every team is different and what motivates them will be different. There are some commonalities, like passion for delivering value to customers, but I'm hesitant to start citing specifics as it's a can of worms to either grossly expanding this section or rattling off banal platitudes. I think this will read a little better in first person. Something like

>I try to ensure that my teams are properly motivated, have all the necessary context, and are granted a high degree of trust and autonomy.